.tool-versions

```diff
@@ -0,0 +1,7 @@
+golang 1.25.0
```

Comment:
question:
are we sure golang 1.25.0 is available in supported SLES version?
just to avoid future issues when submitting to there.